### PR TITLE
Add WAB FATS

### PR DESCRIPTION
**Pull request by sebratton**

### CODE REVIEW

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#build

File: dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class WabAdditionalTests extends AbstractWABTests {
+
+    private static final String SERVLET = "/WebDescriptorServlet";
+    private static final String OUTPUT_WEB_XML_SERVLET = "service: test.wab1.WebXmlMapped";
+
+    private static final String CONFIG = CONFIGS + "testDefault.xml";
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java:31]: This is already a defined constant in the super class.

File: dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+    @BeforeClass
+    public static void startServer() throws Exception {
+        server.setServerConfigurationFile(CONFIG);
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java:35]: You should not need to set the server configuration file to the default `testDefault.xml` here.  The super class `AbstractWABTests.setUp()` method should have set this up with the default `testDefault.xml`  first before this method is called.

**Review by tjwatson — APPROVED** [review]
I have some clean up comments that I would like to see done, but not necessary in this PR.  Could be done in a separate PR.  Up to you what you want to do.

File: dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java (hunk 1 above)
Comment (sebratton) [reply dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java]: Made suggested correction.

File: dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java (hunk 2 above)
Comment (sebratton) [reply dev/com.ibm.ws.app.manager.wab.installer_fat/fat/src/com/ibm/ws/app/manager/wab/installer/fat/WabAdditionalTests.java]: Made suggested correction.

**Comment by sebratton** [comment]
#build

**Comment by sebratton** [comment]
#libbybot

**Pull request merged**